Components/templateWeather.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import {Text, View, StyleSheet} from 'react-native';
+import {Forecast} from './weather.type';
+
+interface Props {
+  forecastElement: Forecast;
+}
+
+export class TemplateWeather extends React.Component<Props> {
+  render() {
+    const forecast: Forecast = this.props.forecastElement;
+    const KtoC: number = 273;
+    const msTokmh: number = 3.6;
+
+    return (
+      <View>
+        <View>
+          <Text style={styles.textTitle}>{forecast.dt_txt}</Text>
+        </View>
+        <View style={styles.caracterictics}>
+          <View style={styles.titleCara}>
+            <View>
+              <Text style={styles.text}>
+                T°Max :{' '}
+                {Math.trunc((forecast.main.temp_max - KtoC) * 100) / 100}
```

Comment:
This should be in a special function right ?
Something like `convertKelvinToCelcius(forecast.main.temp)`